apps/docs/content/guides/platform/orioledb.mdx

```diff
@@ -0,0 +1,118 @@
+
+OrioleDB addresses PostgreSQL's scalability limitations by removing bottlenecks in the shared memory cache under high concurrency and optimizing WAL insertion through a row-level WAL loggin. These improvements are demonstrated by the TPC-C benchmark, which assess real-world database performance. The benchmark was performed on a c7g.metal instance and a 500-warehouse dataset and shows OrioleDB's performance compared to default PostgreSQL heap tables.
+
+![TPC-C (warehouses = 500)](/docs/img/platform/orioledb-tpc-c-500-warehouse.svg)
+
+## Concepts
+
+### Index-organized tables
+
+OrioleDB uses index-organized tables, where table data is stored in the index structure. This design eliminates the need for separate heap storage, reduces overhead and improves lookup performance for primary key queries.
+
+### No buffer mapping
+
+In-memory pages are connected to the storage pages using direct links. This allows OrioleDB to bypass PostgreSQL's shared buffer pool and eliminate the associated complexity and contention in buffer mapping.
+
+### Undo log
+
+Multi-Version Concurrency Control (MVCC) is implemented using UNDO log. The UNDO log stores previous row versions and transaction information, which enables consistent reads while reducing the need for table vacuuming and minimizing bloat in most cases.
+
+### Copy-on-write checkpoints
+
+OrioleDB implements copy-on-write checkpoints to persist data efficiently. This approach writes only modified data during a checkpoint, reducing the I/O overhead compared to traditional PostgreSQL checkpointing and allowing row-level WAL logging.
+
+### Decoupled storage and compute
+
+OrioleDB has experimental support for separating the storage and compute layers. This decoupling allows storage to scale independently of compute resources and enables more flexible resource allocation. This architecture is particularly suited for cloud environments, where storage and compute resources are often provisioned separately.
+
+## Usage
+
+### Creating OrioleDB project
+
+You can get started with OrioleDB by enabling the extension in your Supabase dashboard.
+To get started with OrioleDB you need to [create a new Supabase project](https://supabase.com/dashboard/new/_) and choose `OrioleDB Public Alpha` Postgres version.
+
+![Creating OrioleDB project](/docs/img/platform/orioledb-creating-project.png)
+
+### Creating tables
+
+To create a table using the OrioleDB storage engine just execute the standard `CREATE TABLE` statement. By default it will create a table using OrioleDB storage engine. For example:
+
+```sql
+-- Create a table
+create table blog_post (
+  id int8 not null,
+  title text not null,
+  body text not null,
+  author text not null,
+  published_at timestamptz not null default CURRENT_TIMESTAMP,
+  views bigint not null,
+  primary key (id)
+);
+```
+
+### Creating indexes
+
+OrioleDB tables always have a primary key. If it wasn't defined explicitly, a hidden primary key is created using the `ctid` column.
```

Comment:
even though it was already mentioned, pls add another note here that only btree is supported